datacats/docker.py

```diff
@@ -181,7 +182,8 @@ def web_command(command, ro=None, rw=None, links=None, stream_output=None,
         command=command,
         volumes=binds_to_volumes(binds),
         detach=False,
-        host_config=create_host_config(binds=binds))
+        host_config=create_host_config(binds=binds),
+        entrypoint=entrypoint)
```

Comment:
oh, I googled it never mind: http://serverfault.com/questions/594281/how-can-i-override-cmd-when-running-a-docker-image